flowetl/flowetl/flowetl/operators/staging/sql/create_sightings_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE SCHEMA IF NOT EXISTS reduced;
+CREATE TABLE IF NOT EXISTS reduced.sightings(
+    sighting_date DATE ,  -- Actually partition
+    sub_id bytea NOT NULL, -- change to bytea
+    sighting_id INTEGER NOT NULL,
+    location_id text, --NOT NULL REFERENCES reduced.cell_location_mapping(location_id),  -- change to bytea
+    event_times TIME[],
+    event_types smallint[] -- change to enum
+) PARTITION BY RANGE (sighting_date);
```

Comment:
Move to flowdb, but also please see https://github.com/Flowminder/FlowKit/blob/master/flowdb/bin/build/0030_schema_interactions.sql

and consider whether this should be using the dimension tables in there?